packages/data-helper/src/hooks/useNodeChecks/useNodeChecks.js

```diff
@@ -18,17 +18,16 @@ export const useNodeChecks = (variables, options, queryOptions) => {
 
     if (requiredPermissions) {
         useNodeInfoOptions.getPermissions = useNodeInfoOptions.getPermissions || [];
-        useNodeInfoOptions.getPermissions.push(...requiredPermissions);
+        useNodeInfoOptions.getPermissions = useNodeInfoOptions.getPermissions.concat(requiredPermissions.filter((item) => useNodeInfoOptions.getPermissions.indexOf(item) < 0));
```

Comment:
I think we can just do check for equality to -1, instead of checking for less than 0 for indexOf, but that's just a preference